.env_template

```diff
@@ -0,0 +1,21 @@
+MEDIA_ROOT="$DEV_DIR/media/"
+EPAY_MERCHANT_NUMBER="123"
+EPAY_MD5_SECRET="123"
+COINIFY_API_KEY="123"
+COINIFY_API_SECRET="123"
+COINIFY_IPN_SECRET="123"
+PDF_LETTERHEAD_FILENAME="bornhax.pdf"
+BANKACCOUNT_IBAN="femfladeflødeboller"
+BANKACCOUNT_SWIFTBIC="goldmansachs"
+BANKACCOUNT_REG="1234"
+BANKACCOUNT_ACCOUNT="56789"
+TICKET_CATEGORY_ID="304e9b44-0b48-472d-a05b-963e52614a69"
+SECRET_KEY="muchsecret"
+ALLOWED_HOSTS="127.0.0.1"
+EMAIL_HOST="localhost"
+EMAIL_PORT="22"
+EMAIL_HOST_USER="$USER"
+EMAIL_HOST_PASSWORD=""
+EMAIL_USE_TLS=""
+DEFAULT_FROM_EMAIL="bornhax@localhost"
+ARCHIVE_EMAIL=""
```

Comment:
We already have https://github.com/benjaoming/bornhack-website/blob/dev-improvements/bornhack/settings/env.dist ;)